portfolio/src/main/webapp/script.js

```diff
@@ -28,15 +28,28 @@ function addRandomQuote() {
   quoteContainer.innerText = quote;
 }
 
-/*
- * Adds the data from /data to page
- */
-function addGreetingFromData() {
-    fetch('/data').then(response => response.text()).then((greeting) => {
-        document.getElementById('greeting-container').innerHTML = greeting;
+function addComments() {
+    var response_json = fetch('/data').then(response => response.json());
+    console.log(response_json);
+    response_json.then((comments) => {
+        const commentsListElement = document.getElementById('comments-container');
+        commentsListElement.innerHTML = '';
+        commentsListElement.appendChild(
+            createListElement(comments[0]));
+        commentsListElement.appendChild(
+            createListElement(comments[1]));
+        commentsListElement.appendChild(
+            createListElement(comments[2]));
```

Comment:
Is it possible to convert this to a `for` loop so that you don't need to duplicate code?

Reply:
+1, and also for making sure that it's easier to keep this file and the servlet in sync (e.g. if you change the number of comments)